### PR TITLE
gopass-jsonapi: 1.15.14 -> 1.15.15

### PR DESCRIPTION
**Pull request by doronbehar**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/go/gopass-jsonapi/package.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  subPackages = [ "." ];
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
+    # For ScreenCaptureKit.h
+    apple-sdk_15
```
Comment (doronbehar) [inline pkgs/by-name/go/gopass-jsonapi/package.nix:35]: How to choose the sdk version?

Comment (doronbehar) [reply pkgs/by-name/go/gopass-jsonapi/package.nix:35]: @emilazy Initially I didn't try to add anything Darwin specific, but I got build errors saying:

```
# github.com/kbinani/screenshot
vendor/github.com/kbinani/screenshot/darwin.go:9:10: fatal error: 'ScreenCaptureKit/ScreenCaptureKit.h' file not found
#include <ScreenCaptureKit/ScreenCaptureKit.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

How do you explain this?

Comment (emilazy) [reply pkgs/by-name/go/gopass-jsonapi/package.nix:35]: I guess the issue is that it conditions it on `__ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__ > MAC_OS_VERSION_14_4`, and maybe the undefined `MAC_OS_VERSION_14_4` on our default SDK is defaulting to 0 or something? Anyway you can give it `apple-sdk_14` and omit any `darwinMinVersionHook`.

Comment (doronbehar) [reply pkgs/by-name/go/gopass-jsonapi/package.nix]: OK thanks it worked. I wonder then, what does it mean to use `darwinMinVersionHook` along with a specific `apple-sdk` version? Why do we need the former if we are capable of choosing any of the later?

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by doronbehar** [comment]
> @NixOS/nixpkgs-vet could anyone explain the obscure nix formatting error here?

Can't reproduce the error anymore...

**Pull request merged**